### ISSUE TITLE
Merge riir branch

### ISSUE THREAD

**Issue by fenhl** [wurstmineberg/systemd-minecraft#5]
[The `riir` branch](https://github.com/wurstmineberg/systemd-minecraft/tree/riir), which is a translation of the service into [Rust](https://rust-lang.org/), has been running in production for over 2 years now (since the start of [Zucchini](https://wurstmineberg.de/wiki/renascence#zucchini)), so it should really be merged into the default branch (which should also be renamed from `master` to `main` at that point). Some things that need to be done first:

* The Python module is still being used as a library from other Python code. Need to write a [PyO3](https://docs.rs/pyo3) interface to the new implementation.
* There are still some features missing from the rewrite. Need to check which features those are and port them.
